ext/standard/pack.c

```diff
@@ -55,6 +55,8 @@
 /* Whether machine is little endian */
 char machine_little_endian;
 
+#define IS_LITTLE_ENDIAN (*(unsigned char *)&(uint16_t){1})
+
```

Comment:
Can't you reuse the `WORDS_BIGENDIAN` macro which is defined in Zend/zend_types.h ?

Reply:
Sure can.